getWebpackConfig.js

```diff
@@ -201,6 +201,14 @@ function getWebpackConfig({ apps = [], config = {}, envVars = {}, defineVars = {
             },
           ],
         },
+        {
+          test: /\.m?jsx?$/,
+          loader: 'babel-loader',
+          options: {
+            plugins: ['@babel/plugin-proposal-optional-chaining'],
```

Comment:
https://github.com/primefaces/primevue/issues/874

We need the plugin because the webpack version in the project is pretty old